client/src/app/site/assignments/components/assignment-detail/assignment-detail.component.ts

```diff
@@ -235,7 +235,7 @@ export class AssignmentDetailComponent extends BaseModelContextComponent impleme
         this.tagsObserver = this.tagRepo.getViewModelListBehaviorSubject();
         this.mediafilesObserver = this.mediafileRepo.getViewModelListBehaviorSubject();
 
-        // TODO: why are groups needed here??
+        // Groups are needed, to select entitled groups.
```

Comment:
doesn't they belong into the poll form then?

Reply:
Yes :D